src/button/button.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState, useRef } from "react";
+import Logo from "react:./logo-mark.svg"
+import type { SupportedApplication } from "./button-contributions";
+import classNames from "classnames";
+import { storage_key } from "~storage";
+import { useStorage } from "@plasmohq/storage/hook";
+import React from "react";
+
+export interface GitpodButtonProps {
+  application: SupportedApplication;
+  additionalClassNames?: string[];
+}
+
+export const GitpodButton = ({ application, additionalClassNames }: GitpodButtonProps) => {
+  const [address] = useStorage<string>(storage_key);
```

Comment:
On first install of the extension, no base URL is set, so it breaks by navigating to `undefined`. We can fix it by defaulting to gitpod.io, which we can probably extract to a constant somewhere. 
```suggestion
  const [address] = useStorage<string>(storage_key, DEFAULT_BASE_URL);
```

Reply:
👍  I have inlined it without a constant for since we only use it in one place.